example/posts.js

```diff
@@ -39,6 +41,7 @@ export const PostList = (props) => (
     <List {...props} filter={<PostFilter />}>
         <Datagrid>
             <TextField source="id" />
+            <FileField source="pictures[0]" src="src" title="title" />
```

Comment:
When displaying an image, I'd prefer an `<ImageField>` When displaying the file name with a download link, I'd prefer a `<FileField>`.